.travis.yml

```diff
@@ -61,7 +61,7 @@ install:
   - git clone https://github.com/openresty/openresty.git ../openresty
   - git clone https://github.com/openresty/openresty-devel-utils.git
   - git clone https://github.com/simpl/ngx_devel_kit.git ../ndk-nginx-module
-  - git clone https://github.com/openresty/lua-nginx-module.git ../lua-nginx-module
+  - git clone -b fix_process_type_returns_master https://github.com/spacewander/lua-nginx-module.git ../lua-nginx-module
```

Comment:
@thibaultcha Remember changing this before merging ;)

Reply:
@agentzh Yeah I had noted that, no worries. Thanks for the reminder :)